index.rst

```diff
@@ -40,6 +40,11 @@ fundamental array data structure), see `scipy.org <http://www.scipy.org/>`__.
 
 .. raw:: html
 
-   <div style="padding-top: 40px;text-align:center;">
-     <script type="text/javascript" src="http://www.ohloh.net/p/4894/widgets/project_partner_badge.js"></script>
+   <div style="padding-top:40px; text-align:center; margin:0px auto;">
+     <div style="display: inline-block; float:left; margin-left:1em;">
+       <a href ="https://www.flipcause.com/secure/cause_pdetails/MzUwMQ==" target="_blank" style="background:#f5cb33; border-radius:0px 0px 0px 0px;font-weight:normal; font-family:Arial, Helvetica, sans-serif;border:none;box-shadow:none; clear: both;display: block; width:150px;height:34px; line-height:2.8; font-size:13px;text-align:center; cursor:pointer;color:#000000;text-decoration: none; z-index:1">Donate to Numpy</a>
+     </div>
+     <div style="display: inline-block;">
+       <script type="text/javascript" src="http://www.ohloh.net/p/4894/widgets/project_partner_badge.js"></script>
```

Comment:
No, I mean change the `http` in `http://www.ohloh.net/p/4894/widgets/project_partner_badge.js` to `https`